library/src/test/java/com/datatorrent/lib/appdata/snapshot/AppDataSnapshotServerTagsSupportTest.java

```diff
@@ -0,0 +1,67 @@
+package com.datatorrent.lib.appdata.snapshot;
```

Comment:
Travis build is failing because correct license is not added at the top of the file. Please see other files in Malhar and copy the license header over. Also make sure that the year stated in the license is this year.